logins-api/android/library/src/test/java/org/mozilla/sync15/logins/DatabaseLoginsStorageTest.kt

```diff
@@ -0,0 +1,25 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+package org.mozilla.sync15.logins
+
+import org.junit.rules.TemporaryFolder
+import org.junit.Rule
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+class DatabaseLoginsStorageTest: LoginsStorageTest() {
+    @Rule
+    @JvmField
+    val dbFolder = TemporaryFolder()
+
+    override fun createTestStore(): LoginsStorage {
```

Comment:
Can we run this against an in-memory database?

That's actually a general thing I'd like to track -- making sure that we're clear on in-memory stores and correctly passing SQLite URIs down the stack.  That'd let us use in-memory stores and concurrently access them, etc.

Reply:
I think everything will *probably* work if we pass in ":memory:" as the file.

Reply:
It certainly does not! I'll file a bug.

Reply:
Actually, it does work. The tests assume lock/unlock doesn't discard everything, which is sort of guaranteed by the API, so in that sense it doesn't. Filed #257